soc/arm/silabs_exx32/common/soc.c

```diff
@@ -189,6 +200,18 @@ static int silabs_exx32_init(const struct device *arg)
 	/* disable interrupts */
 	oldLevel = irq_lock();
 
+#ifdef CONFIG_SOC_GECKO_DEV_INIT
+	CHIP_Init();
+	sl_board_preinit();
```

Comment:
this call seems to be initializing GPIO clocks. This must be done by the GPIO driver.

Reply:
Moved initializing GPIO clocks to GPIO driver.